profile/README.md

```diff
@@ -1,23 +1,37 @@
-## Hey, this is us 👋
```

Comment:
@Rohitrky2021 I somewhere agree with @ayan-biswas0412 . The initial looks better, however we can add contact us and update community content:

============

## Hey, this is us 👋
</br>
<img src="https://raw.githubusercontent.com/CircuitVerse/CircuitVerse/master/app/assets/images/cvlogo.svg" alt="The CircuitVerse logo" width="750"/>
</br>

[CircuitVerse](https://circuitverse.org) is a free, open-source platform which allows users to construct digital logic circuits online. We also offer the [Interactive Book](https://learn.circuitverse.org) which teaches users on the fundamentals of modern, digital circuits. Please also see our [documentation](https://docs.circuitverse.org) or [GitHub Wiki](https://github.com/CircuitVerse/CircuitVerse/wiki/).

## 🏂 Haven't tried CircuitVerse yet ?

Here are few things which we have developed in last couple of years.
 - Try [CircuitVerse](https://circuitverse.org/), which is an easy to use digital circuit simulator.
 - Checkout the [documentation](https://docs.circuitverse.org/#/) to get started.
 - For learning basics of digital circuit design here is our [Interactive book](https://learn.circuitverse.org/).

## 🍿 Community-Driven Development

CircuitVerse is built in the open, and we welcome contributions from the community. If you'd like to help us make CircuitVerse better, here are some ways to get involved:

- Join us on [GitHub](https://github.com/CircuitVerse/CircuitVerse) and submit bug reports or feature requests.
- Contribute to the platform by submitting pull requests with code changes or new features.
- Participate in our community forums and chat with other CircuitVerse users.

## 📧 Contact Us

If you have any questions or feedback, we'd love to hear from you. You can join our [slack](https://circuitverse-team.slack.com/ssb/redirect). You can reach us via email at [support@circuitverse.org](mailto:support@circuitverse.org), or connect with us on [Twitter](https://twitter.com/CircuitVerse).

Made with 🖤
🙇‍♂️ 🎤 ⬇️

Reply:
@vedant-jain03  ok sir ! ya i agree